### PR TITLE
Add form validation

### PR DESCRIPTION
**Pull request by teke85**
This Pull Request is based on the Form Validation (JavaScript Version) Project requirement and it fulfils the following objectives.

# General requirements
To make sure that there are no linter errors.
To make sure that we use the correct GitHub flow.
To make sure that we document our work in a professional way.

# Project requirements

To implement a simple validation:
The content of the email field must be in lowercase.
The following interactions should be implemented.
- When the user submits the form, we should check if the email characters are in lower case.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

Collaborators
@tashisrael @Okechukwu-muokwugwo

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -210,3 +210,32 @@ window.onclick = (event) => {
...
+  const { email } = form.elements;
+  if (!checkEmail(email)) {
+    email.setCustomValidity('Please enter a valid email address and in lowercase letters.');
+    emailError.textContent = 'Please enter a valid email address and in lowercase letters.';
+  } else {
+    email.setCustomValidity('');
+    emailError.textContent = '';
+    emailError.className = 'error';
+    form.submit();
+  }
```
Comment (roshan-bajgain) [inline script.js]: - [ ] If the validation is not OK, you should show an error message to the user near the `Get in Touch` button. Please implement it.

**Review by mwanawabangona — APPROVED** [review]
Hi @teke85 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job getting approved on your first try!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @teke85, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
- [ ] I would suggest you please don't delete the pull request until the approval, use the same pull request because we need to be consistent on the first review.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ennio86** [comment]
Hi @teke85 - please ignore the approval message and work on the changes requested by @roshan-bajgain. 🙏
@mwanawabangona - please make sure to pay close attention to review guidelines when performing CCR. 🙏

**Comment by teke85** [comment]
> Hi @teke85,
> 
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
> 
> ## Highlights
> * No linter errors. ✔️
> * GitHub flow is followed. ✔️
> * PR has a good title and summary. ✔️
> 
> ## Required Changes ♻️
> * [ ]  I would suggest you please don't delete the pull request until the approval, use the same pull request because we need to be consistent on the first review.
>   _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (teke85) [reply script.js]: @roshan-bajgain thank you for your observation. I have implemented the suggested functionality. When validation is not ok an error is shown above the okay button.

Portfolio (9)

**Comment by teke85** [comment]
Hi @roshan-bajgain  , Thank you for your comment however I didn't delete the first pull request. The first code reviewer who reviewed this project didn't leave a comment and because of that I reached out to microverse and I was advised to submit a new code review. Below is the correspondence I made with [Ennio Poscia] https://microverse.zendesk.com/hc/en-us/requests/202627 regarding the pull request issue.
Project-Review-Submission-–-Microverse

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**